rust_src/src/character.rs

```diff
@@ -26,3 +26,17 @@ fn characterp(object: LispObject, _ignore: LispObject) -> LispObject {
 fn char_or_string_p(object: LispObject) -> LispObject {
     LispObject::from_bool(object.is_character() || object.is_string())
 }
+
+/// Convert the byte CH to multibyte character.
+#[lisp_fn]
+fn unibyte_char_to_multibyte(ch: LispObject) -> LispObject {
+    ch.is_character_or_error();
+    let mut c = ch.as_fixnum().unwrap() as u32;
+    if c >= 0x100 {
+        unsafe {
+            error("Not a unibyte character: %d".as_ptr(), c);
```

Comment:
I believe this should be `error("Not a unibyte character, %d\0".as_ptr(), c)`, as Rust does not null-terminate it's string literals automatically.